pkg/steps/artifacts.go

```diff
@@ -316,7 +316,9 @@ touch /tmp/done
 echo "Waiting for artifacts to be extracted"
 while true; do
 	if [[ ! -f /tmp/done ]]; then
-		echo "Artifacts extracted"
+		echo "Artifacts extracted, will terminate after 60s"
```

Comment:
Why do you say 60s here?

Reply:
If you want to delay shutdown, it would be better to have a prestop hook.

Reply:
b/c I'm an idiot, had `sleep 60` here first, then shortened it to 30s and forgot to fix the message :)

I'll look into prestop hooks (I'm not familiar with them) if you want me to, but this is really just a band-aid so that we don't hit the problem that often.

I'd like to eventually fix this properly by implementing some synchronization so that the artifacts container never terminater before all ci-operator instances that wait for the pod and extract artifacts finish.